styles/stylesheet.css

```diff
@@ -0,0 +1,414 @@
+body {
+
+  margin:0;
+
+}
+
+div {
+  /*
+  min-height: 1000px;
+  margin-bottom: 100px;
+  */
+}
+
+header {
+  background-color: #2c9ab7;
+  background: #000000 url(../images/header-image.jpg) 100% 25% no-repeat;
+  margin:0;
+  padding:200px;
+}
+
+footer {
+
+ right: 0;
+ bottom: 0;
+ left: 0;
+ padding-top: 50px;
+ padding-bottom:50px;
+ background-color: #083b5f;
+ text-align: center;
+}
+
+section {
+
+  margin-left:255px;
+  margin-right:255px;
+  margin-top: 50px;
+  margin-bottom:50px;
+}
+
+.sectionblog {
+  padding-bottom:50px;
+}
+
+.mainimage {
+  float:left;
+  padding-right:50px;
+}
+.dogimage {
+  width:100%
+}
+
+.dogimage-small {
+  width:8%
+}
+.clearfix {
+  overflow: auto;
+  margin-left: 40px;
+  margin-right: 70px;
+}
+.clearfix-form {
+    overflow: auto;
+    margin-left: 20px;
+    margin-right: 300px;
+}
+.mainp {
+  font-family: 'Work Sans', sans-serif;
+  font-size: x-large;
+
+}
+.mainh1 {
+  margin-top: 0px;
+  font-size: xx-large;
+  font-family: 'Neuton', serif;
+  color: darkslategrey;
+
+}
+
+.dogsh1 {
+  font-size: xxx-large;
```

Comment:
I would ignore the issue with the validator for now as this should not be creating an error. However I would advise against font size keywords and instead us em or rem for font-size. These keywords can vary from browser to browser and explicitly stating the size will produce more accurate results.